### PR TITLE
Update rfc-template.md

### PR DESCRIPTION
**Pull request by eldadfux**
1. Added a few more sections to the design process to make it a bit more obvious and make sure we cover all required topics.
2. Removed extra questions from the problem statement  to make it easier to describe.

### CODE REVIEW

**Review by lohanidamodar — APPROVED** [review]
Looks great!

File: rfc-template.md [hunk 1]
```diff
@@ -48,33 +41,93 @@ This is the technical portion of the RFC. Explain the design in sufficient detai
...
+-->
+
+### Breaking Changes
+
+<!--
+Do we break any API or SDK backward compatibility?
+If possible, explain what actions we can take to avoid that.
+-->
+
+### Relaibility (Tests & Benchmarks)
```
Comment (christyjacob4) [inline rfc-template.md]:
```suggestion
### Reliability (Tests & Benchmarks)
```

File: rfc-template.md [hunk 2]
```diff
@@ -48,33 +41,93 @@ This is the technical portion of the RFC. Explain the design in sufficient detai
...
+
+#### Benchmarks
+
+<!-- Explain how we will benchmark the new feature. -->
+
+#### Tests (UI, Unit, E2E)
+
+<!-- Explain how we will test the new feature. -->
+
+### Documentatation & Content
```
Comment (christyjacob4) [inline rfc-template.md]:
```suggestion
### Documentation & Content
```

File: rfc-template.md [hunk 3]
```diff
@@ -48,33 +41,93 @@ This is the technical portion of the RFC. Explain the design in sufficient detai
...
+
+<!-- Explain how we will scale this new feature. -->
+
+#### Benchmarks
+
+<!-- Explain how we will benchmark the new feature. -->
+
+#### Tests (UI, Unit, E2E)
+
+<!-- Explain how we will test the new feature. -->
```
Comment (christyjacob4) [inline rfc-template.md]:
```suggestion
<!-- 
Explain how we will test the new feature. 
You can use "N/A" if this section is not relevant to your proposal.
-->
```

Can we also leave a comment telling what they need to do if a section is not relevant for them ?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: rfc-template.md [hunk 4]
```diff
@@ -48,33 +41,93 @@ This is the technical portion of the RFC. Explain the design in sufficient detai
...
+### Breaking Changes
+
+<!--
+Do we break any API or SDK backward compatibility?
+If possible, explain what actions we can take to avoid that.
+-->
+
+### Relaibility (Tests & Benchmarks)
+
+#### Scaling
```
Comment (christyjacob4) [inline rfc-template.md:89]:
```suggestion
* #### Scaling
```

File: rfc-template.md [hunk 5]
```diff
@@ -48,33 +41,93 @@ This is the technical portion of the RFC. Explain the design in sufficient detai
...
+
+<!-- Explain how we will scale this new feature. -->
+
+#### Benchmarks
```
Comment (christyjacob4) [inline rfc-template.md:93]:
```suggestion
* #### Benchmarks
```

File: rfc-template.md [hunk 6]
```diff
@@ -48,33 +41,93 @@ This is the technical portion of the RFC. Explain the design in sufficient detai
...
+
+<!-- Explain how we will benchmark the new feature. -->
+
+#### Tests (UI, Unit, E2E)
```
Comment (christyjacob4) [inline rfc-template.md:97]:
```suggestion
* #### Tests (UI, Unit, E2E)
```

**Review by Meldiron — APPROVED** [review]
Looks good to me. Happy to test it in the future 😊

File: rfc-template.md (hunk 4 above)
Comment (eldadfux) [reply rfc-template.md:89]: What is the idea behind the list? We already use a sub-title. Actually, i'm not sure this is correct accessibility wise.

**Comment by eldadfux** [comment]
Gonna merge this without the bullet point for now. @christyjacob4 let me know if you want to discuss this later.

**Pull request merged**